pkg/issuer/acme/acme.go

```diff
@@ -38,8 +37,6 @@ import (
 // certificates from any ACME server. It supports DNS01 and HTTP01 challenge
 // mechanisms.
 type Acme struct {
-	issuer v1.GenericIssuer
```

Comment:
Instead of storing the issuer in these structs, we now just pass the issuer as an argument to Setup().